src/controllers/change.answers.controller.ts

```diff
@@ -0,0 +1,19 @@
+import {NextFunction, Request, Response} from "express";
+import {addToObjectionSession} from "../services/objection.session.service";
+import {Session} from "ch-node-session-handler";
+import {STRIKE_OFF_OBJECTIONS} from "../model/page.urls";
+import {CHANGE_ANSWER_KEY} from "../constants";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  const session: Session | undefined = req.session as Session;
+  if (session) {
+    const query: string | undefined = req.query["changePage"] as string;
```

Comment:
Would 'changePage' be a better name for this constant/variable? 'query' is the operation we seem to be performing on the request to get the name of the page to change to...

Reply:
Chnaged the name but kept query as part of it because we test whether it is truthy or not.